string destination
bool isAsciiMapSave
float32 xRange
float32 yRange
float32 intensityScale
---
bool success
